types/wlr_output.c

```diff
@@ -18,6 +18,8 @@
 
 #define OUTPUT_VERSION 3
 
+#define DEFAULT_PRESENT_CLOCK CLOCK_MONOTONIC
```

Comment:
-1

Reply:
What do you mean?

Reply:
This doesn't need to be abstracted, just use CLOCK_MONOTONIC explicitly.

Reply:
There's more to come.

Reply:
Okay, but I'm probably not going to change my mind.